### PR TITLE
Associate kms key

### PR DESCRIPTION
**Pull request by Paramadon**
# Description of the issue
_Describe the problem or feature in addition to a link to the issues._

# Description of changes
_How does this change address the problem?_

# License
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

# Tests
_Describe what tests you have done._

# Requirements
_Before commit the code, please do the following steps._
1. Run `make fmt` and `make fmt-sh`
2. Run `make lint`

### CODE REVIEW

File: translator/translate/logs/logs_collected/files/collect_list/ruleLogFilters.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT
 
-package collect_list
+package collectlist
```
Comment (Paramadon) [inline translator/translate/logs/logs_collected/files/collect_list/ruleLogFilters.go:4]: Needed to change package name due to lint issues

**Review by TravisStark — APPROVED** [review]

File: plugins/outputs/cloudwatchlogs/cloudwatchlogs_test.go [hunk 2]
```diff
@@ -77,7 +77,7 @@ func TestCreateDestination(t *testing.T) {
 				pusherStopChan: make(chan struct{}),
 				cwDests:        make(map[pusher.Target]*cwDest),
 			}
-			dest := c.CreateDest(testCase.cfgLogGroup, testCase.cfgLogStream, testCase.cfgLogRetention, testCase.cfgLogClass, testCase.cfgTailerSrc).(*cwDest)
+			dest := c.CreateDest(testCase.cfgLogGroup, testCase.cfgLogStream, "", testCase.cfgLogRetention, testCase.cfgLogClass, testCase.cfgTailerSrc).(*cwDest)
```
Comment (nathalapooja) [inline plugins/outputs/cloudwatchlogs/cloudwatchlogs_test.go:80]: Add a unit test when kms id is not empty and validate the `dest ` for the key
